### PR TITLE
Removing native js dependency

### PR DESCRIPTION
**Pull request by nebojsa94**
This PR removes keccak and secp256k1 dependencies and instead uses ethereum-cryptography package that doesn't require native dependency compiling

### CODE REVIEW

File: src/account.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
 const ethjsUtil = require('ethjs-util')
+const {
+  privateKeyVerify,
+  publicKeyCreate,
+  publicKeyVerify,
+  publicKeyConvert,
+} = require('ethereum-cryptography/shims/hdkey-secp256k1v3')
```
Comment (alcuadrado) [inline src/account.ts:7]: See: https://github.com/ethereumjs/ethereumjs-util/pull/259/files#r441088580

File: src/hash.ts [hunk 2]
```diff
@@ -12,9 +12,23 @@ import { assertIsString, assertIsBuffer, assertIsArray, assertIsHexString } from
...
+      return k256(a)
+    }
+    case 384: {
+      return keccak384(a)
+    }
+    case 512: {
+      return keccak512(a)
+    }
+    default: {
+      throw new Error(`Invald algorithm: keccak${bits}`)
```
Comment (alcuadrado) [inline src/hash.ts:29]: See: https://github.com/ethereumjs/ethereumjs-util/pull/259/files#r441089259

File: src/signature.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import * as secp256k1 from 'secp256k1'
+const { sign, publicKeyConvert } = require('ethereum-cryptography/shims/hdkey-secp256k1v3')
```
Comment (alcuadrado) [inline src/signature.ts:1]: See: https://github.com/ethereumjs/ethereumjs-util/pull/259/files#r441088580

**Comment by holgerd77** [comment]
@nebojsa94 oh sorry, I merged in another PR https://github.com/ethereumjs/ethereumjs-util/pull/248, didn't expect this to conflict with this one. 😕  Could you give this a final update, I will directly merge this in afterwards.

@alcuadrado thanks for the review and the additional context. Should we directly publish this as `v7.0.3` then I assume?

**Comment by alcuadrado** [comment]
> Should we directly publish this as `v7.0.3` then I assume?

Yep, this is the v7 version of these changes, so `v7.0.3` should do.

**Pull request merged**